include/beast/http/impl/basic_headers.ipp

```diff
@@ -205,6 +206,18 @@ basic_headers(FwdIt first, FwdIt last)
 }
 
 template<class Allocator>
+std::size_t
+basic_headers<Allocator>::
+count(boost::string_ref const& name) const
+{
+    auto const it = set_.find(name, less{});
+    if(it == set_.end())
+        return 0;
+    auto const last = set_.upper_bound(name, less{});
+    return static_cast<std::size_t>(std::distance(it, last));
+}
+
+template<class Allocator>
 auto
 basic_headers<Allocator>::
 find(boost::string_ref const& name) const ->
```

Comment:
Since we're now set up for duplicate entries, I just wanted to make sure we're happy that `find()` always returns the first entry (inserted in time sequence).  I think that's the behavior we'll see (since `insert()` always appends), and I think it's a reasonable behavior.  I just thought I'd mention it...